guides/common/modules/proc_deleting-a-vm-on-google-gce.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Deleting_a_VM_on_Google_GCE_{context}"]
+= Deleting a VM on Google GCE
+
+You can delete VMs running on Google GCE from within {Project}.
```

Comment:
Maybe `on your {ProjectServer}` would be safer than `from within {Project}` ?
I'm not sure how that would behave in automated translation :))

Reply:
Applied.